### DISCUSSION TITLE
com.openai.unity 8.0.1

### DISCUSSION THREAD

**Discussion by StephenHodgson** [RageAgainstThePixel/com.openai.unity#247] [Announcements]
## What's Changed
- Fixed Thread.Run.Status enum ordering
- Fixed sample scene

**Full Changelog**: https://github.com/RageAgainstThePixel/com.openai.unity/compare/8.0.0...8.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/RageAgainstThePixel/com.openai.unity/releases/tag/8.0.1'>com.openai.unity 8.0.1</a>.</em>

**Comment by ElisMia** [comment]
Hey, @StephenHodgson ! Thank you for update of the package! It's really great to have v2 version. 

Could you please give a tip? Here is how I get "vector id" of "file id" my assistant has. Is there some better way to do it?  
```csharp
 var storeList = await api.VectorStoresEndpoint.ListVectorStoresAsync();
           
            foreach (var vectorStore in storeList.Items)
            {
                var vectorStoreId = vectorStore.Id;
                var filesResponse = await api.VectorStoresEndpoint.ListVectorStoreFilesAsync(vectorStoreId);
                var files = filesResponse.Items;
                if (files.Any(file => file.Id == fileId))
                {
                    // This vector store contains your file
                    // Now you have the vector store id (vectorStoreId) that contains your file
                }
            }
```

**Reply by StephenHodgson** [reply]
Please open new discussion thread in https://github.com/RageAgainstThePixel/com.openai.unity/discussions/categories/q-a

**Reply by StephenHodgson** [reply]
I believe that this can be accessed from [AssistantResponse.ToolResources.FileSearchResources](https://rageagainstthepixel.github.io/OpenAI-DotNet/api/OpenAI.FileSearchResources.html)
